### PR TITLE
Schema:dump and schema:load commands

### PR DESCRIPTION
**Pull request by detook**
MySQL only implementation of #109. After your review I'll continue with other adapters.

### CODE REVIEW

**Comment by adamski** [comment]
Anyone know if this fork is compatible with recent additions to the master branch i.e. 4.x? 

The new functionality for using Phinx via web interface is just what I am trying to implement, currently via php exec(). However also need to use schema:load to automate creation of new databases. Described here: https://github.com/robmorgan/phinx/blob/0.4.x-dev/docs/commands.rst

**Comment by detook** [comment]
@adamski, not yet. I'm going to look at 0.4.x and new schema type support this weekend.

**Comment by adamski** [comment]
Thank you @detook, your efforts are very much appreciated!

**Comment by oniric85** [comment]
:+1:

File: src/Phinx/Console/Command/Migrate.php [hunk 1]
```diff
@@ -1,94 +1,95 @@
...
-        // run the migrations
-        $start = microtime(true);
-        $this->getManager()->migrate($environment, $version);
-        $end = microtime(true);
-        
-        $output->writeln('');
-        $output->writeln('<comment>All Done. Took ' . sprintf('%.4fs', $end - $start) . '</comment>');
-    }
-}
+<?php
```
Comment (shadowhand) [inline src/Phinx/Console/Command/Migrate.php]: why was this entire file changed?

**Comment by shadowhand** [comment]
Please create a new pull request against `0.4.x-dev` branch. Thanks!

**Pull request closed without merging**